composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "prooph/psb-zeromq-producer",
+    "description": "ZeroMQ Message Producer for Prooph Service Bus",
+    "homepage": "https://github.com/bweston92/psb-zeromq-producer",
+    "license": "BSD-3-Clause",
+    "authors": [
+        {
+            "name": "Bradley Weston",
+            "email": "me@bweston.me"
+        }
+    ],
+    "keywords": [
+        "prooph",
+        "CQRS",
+        "messaging",
+        "zeromq"
+    ],
+    "require": {
+        "php": ">=5.5",
+        "ext-zmq": "*"
+    },
+    "require-dev": {
+        "prooph/service-bus": "dev-master",
```

Comment:
We've decided to [explicitly define dependencies](https://github.com/prooph/event-store-bus-bridge/pull/6#issuecomment-141636819) because not all versions of prooph/service-bus are suitable for the producer. Only `prooph/service-bus : ^4.2`. I've created issues to also correct this for the http and bernard producer. @bweston92 Please adjust the composer.json of the zeromq producer.